src/content/docs/distribute/flatpak.mdx

```diff
@@ -61,42 +115,11 @@ If you are developing in a container that shares files with the host, you can in
 Any command with a `(host)` label can be executed on your host, while commands without this label must be executed inside the container.
 `(host)` commands must be executed from inside the flatpak directory on your host.
```

Comment:
I wonder if we should remove all the container vs host talk and just assume everything's on the host to not add confusion 🤔